### DISCUSSION TITLE
state.IsFocused is always true for first option on initial render

### DISCUSSION THREAD

**Discussion by rokin04** [JedWatson/react-select#4662] [General]
Hi All,

I'm passing defaultValue prop to have initially selected option , on initial render state.isSelected is true for defaultValue but state.isFocused is not, is there any workaround for this ?

This is happening when the defaultValue is changed from controls in the storybook, not in the actual code.

**Comment by ebonow** [comment]
Greetings @rokin04 ,

What is the initial value you are passing to defaultValue and can you explain by what you mean 
> "when the defaultValue is changed"

defaultValue is only applied at the time of render so any changes to it would not be reflected in the control
